neural_modelling/src/synapse_expander/delay_expander.mk

```diff
@@ -0,0 +1,7 @@
+APP = delay_expander
+BUILD_DIR = build/
+SOURCES = rng.c param_generator.c connection_generator.c delay_expander.c
+
+include ../common.mk
+
```

Comment:
I think @Christian-B was attempting to avoid the use of ../ in makefiles in a previous PR - can it be done here?